src/printer/components/toast.js

```diff
@@ -50,7 +50,7 @@ function show({ title, message, type, onClose }) {
   });
 
   const timeout = timeouts[type];
-  if (timeout)
+  if (timeout && autoClose)
```

Comment:
How did this timeout work before? Isn't it going to break something now that `autoClose` is required for the timeout to work?

Reply:
I made autoClose be True by default, so for most scenarios, it should be a non factor.
If the first statement holds true, then the only time this could break is with my new custom warning that is not supposed to close automatically, which it doesn't, so i'd call that working as intended?

I am not really sure what are you getting at here, so please clarify, if i didn't get your point

Reply:
Oh I see now, I didn't connect the default arguments in my mind.

Nevermind, it looks good to go 👍